docs/user/discover.asciidoc

```diff
@@ -275,8 +270,17 @@ image:images/discover-maps.png[Map containing documents]
 [[share-your-findings]]
 === Share your findings
 
-To share your findings with a larger audience, click *Share* in the *Discover* toolbar. For detailed information about the sharing options, refer to <<reporting-getting-started,Reporting>>.
+To share your findings with a larger audience, click *Share* in the *Discover* toolbar.
+For detailed information about the sharing options, refer to <<reporting-getting-started,Reporting>>.
+
+[float]
+[[alert-from-Discover]]
+=== Create alert rules
```

Comment:
"Alert rules" isn't a term we've used elsewhere in the docs, so I'd suggest:
```suggestion
=== Create rules for alerting
```
or
> === Generate alerts